circtpanamabinding/includeFunctions.txt

```diff
@@ -104,6 +104,7 @@ firrtlAttrGetNameKind
 firrtlAttrGetRUW
 firrtlAttrGetMemoryInit
 firrtlAttrGetMemDir
+firrtlAttrGetIntegerFromString
```

Comment:
Please remove the original C-API call and usages to avoid dead code.

Reply:
`mlirIntegerAttrGet` is still used in elsewhere so it can't be removed.

Reply:
Is possible to align them to `firrtlAttrGetIntegerFromString`

Reply:
For small integers (less than 64 bits), they don't need to be parsed from a string with the hack, aligning them will downgrade the performance.